source/kraken/configuration.cpp

```diff
@@ -165,6 +167,10 @@ bool Configuration::is_realtime_add_enabled() const{
     return this->vm["GENERAL.is_realtime_add_enabled"].as<bool>();
 }
 
+bool Configuration::enable_realtime_add_trip() const{
+    return this->vm["GENERAL.enable_realtime_add_trip"].as<bool>();
```

Comment:
It would probably be safer to (untested):
```c++
return (is_realtime_add_enabled() && this->vm["GENERAL.enable_realtime_add_trip"].as<bool>());
```

Reply:
In this part of code, we simply load params from config. We don't want to insert logic. Only load the _raw value_ of the flag.

Reply:
Also: we will probably use that when triggering re-indexing and all realtime ptref creations, and we want to activate them only when the 3 params are `true`.
I want to secure it that way, because it doesn't seem to make any sense to handle the case where only `enable_realtime_add_trip=true` and the other false.